src/raw/mod.rs

```diff
@@ -948,59 +1059,76 @@ impl<T> RawIterRange<T> {
     ///
     /// The start offset must be aligned to the group width.
```

Comment:
This is the comment I was trying to point out is out-of-date now (there is no longer a "start offset"?)